assets/shaders/identity.vert.glsl

```diff
@@ -1,16 +1,14 @@
-#version 120
+#version 330 core
 
-// no-transformation texture mapping vertex shader
+// Input vertex position
+layout(location = 0) in vec4 vertex_position;
 
-// the position of this vertex
-attribute vec4 vertex_position;
-
-// interpolated texture coordinates sent to fragment shader
-varying vec2 tex_position;
+// Output to fragment shader
+out vec2 tex_position;
 
 void main(void) {
-	gl_Position = vertex_position;
+    gl_Position = vertex_position;
```

Comment:
afaik you don't need this statement since `gl_Position` by definition contains the current vertex position already. You can remove that line.

```suggestion
```